sdk/src/feature_set.rs

```diff
@@ -463,7 +464,7 @@ pub mod preserve_rent_epoch_for_rent_exempt_accounts {
 }
 
 pub mod enable_bpf_loader_extend_program_ix {
-    solana_sdk::declare_id!("8Zs9W7D9MpSEtUWSQdGniZk2cNmV22y6FLJwCx53asme");
+    solana_sdk::declare_id!("JC5GB7HH2BD5NuFfiWBV76XAnsvEpPz8LgPEvVb1tn8Q");
```

Comment:
This rekeys the `enable_bpf_loader_extend_program` feature gate to a keypair in our custody.